tests/kernel_performance/install_qatestset.pm

```diff
@@ -40,7 +40,6 @@ sub setup_environment {
     my $ver_cfg           = get_required_var('VER_CFG');
     my $ver_path          = "/root";
```

Comment:
This variable is not used in this function, is it possible to remove it? Thanks!

Reply:
removed,thanks.